pkg/api/controller.go

```diff
@@ -58,6 +58,13 @@ func NewController(config *config.Config) *Controller {
 	var controller Controller
 
 	logger := log.NewLogger(config.Log.Level, config.Log.Output)
+
+	if config.Cluster != nil {
+		// add global context of address:port for the instance's loggers only if
+		// scale-out cluster mode is enabled
+		host := fmt.Sprintf("%s:%s", config.HTTP.Address, config.HTTP.Port)
```

Comment:
This is not going to work right? Since you are likely only listening on 0.0.0.0:9000 which is same for every member. You really want to find that entry in the cluster memberlist that belongs to this node.

Reply:
But this log is super-important